v2/session/common-customizations/sessions/multiple-api-endpoints.mdx

```diff
@@ -112,7 +112,101 @@ Whilst the `cookieDomain` can start with a leading `.`, the value of the `apiDom
 For local development, you should not set the `cookieDomain` to an IP address based domain, or `.localhost` - browsers will reject these cookies. Instead, you should [alias `localhost` to a named domain and use that](https://superuser.com/questions/152146/how-to-alias-a-hostname-on-mac-osx). 
 :::
 
-## Step 2) Frontend config
+## Step 2) Set Older Cookie Domain in the Backend Config
+
+:::caution
+If `olderCookieDomain` isn't set, users with older sessions will be locked out until the `olderCookieDomain` is set correctly or they clear their cookies.
```

Comment:
define what locked out means -> the session refresh endpoint will throw a 500 error with a message.